debezium-connector-mongodb/pom.xml

```diff
@@ -216,6 +216,18 @@
                 <groupId>io.debezium</groupId>
                 <artifactId>debezium-schema-generator</artifactId>
                 <version>${project.version}</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>${project.groupId}</groupId>
+                        <artifactId>${project.artifactId}</artifactId>
+                        <version>${project.version}</version>
+                    </dependency>
+                    <dependency>
+                        <groupId>org.apache.kafka</groupId>
+                        <artifactId>connect-api</artifactId>
+                        <version>${version.kafka}</version>
+                    </dependency>
+                </dependencies>
```

Comment:
That's not needed any longer as per the commit I've pushed (same for the other modules).

Reply:
Removed everywhere